domains/windows.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "my website uwu",
+  "repo": "https://github.com/windowscmd",
+  "owner": {
+    "username": "windowscmd",
+    "email": "me@astriddev.me"
+  },
+    "record": {
+      "CNAME": "astridw.netlify.app"
+    }
+  }
```

Comment:
Remove this }

Reply:
why remove?

Reply:
this still hasn't been done, its still invalid JSON...